world_frame: "world"

# points of the input pointcloud, closer than this size, are discarded
filtering_leaf_size: 0.5 # meters

arena_box:
  offset: # offset of the box center relative to the world origin
    x: 0 # meters
    y: 0 # meters
    z: 5 # meters
  size:
    x: 100 # meters
    y: 60 # meters
    z: 10 # meters

# points inside the box, defined by *uav_exclude_box*, relative to the sensor, are discarded
# this is used to filter out the MAV itself from the pointcloud
exclude_box:
  offset: # offset of the box center relative to the sensor origin
    x: 0 # meters
    y: 0 # meters
    z: -0.3 # meters
  size:
    x: 1 # meters
    y: 1 # meters
    z: 1 # meters

keep_pc_organized: true
